portfolio/src/main/webapp/script.js

```diff
@@ -17,10 +17,12 @@
  */
 function addRandomFact() {
   const facts =
-      ['I have a scar on the back of my head from when I fell backwards out of a chair when I was in 5th grade.',
-       'I backpacked through Portugal with a Catalan Troop.', 
-       'I have 6 pets, 5 cats and 1 dog.', 
-       'I forgot I was in the orchestra pit for "South Pacific."'];
+    ['I have a scar on the back of my head from when I fell backwards out of a chair when I was in 5th grade.',
+      'I backpacked through Portugal with a Catalan Troop.',
+      'I have 6 pets, 5 cats and 1 dog.',
+      'I forgot I was in the orchestra pit for "South Pacific."',
+      'I still have a baby tooth.',
+      "I've baked chocololate chip cookies for my entire highschool more than once."];
```

Comment:
The style guide says to use single quotes everywhere in JavaScript, so probably just escape the internal quote. It also says you can use template strings, so maybe look into that: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
I thought I made it so that each new indent was two spaces. Is it possible that it is just super indented?

Reply:
Oh, so the quotation marks should line up with the first line? Ok.